ios/RNIterableAPI/ReactIterableAPI.swift

```diff
@@ -213,6 +213,23 @@ class ReactIterableAPI: RCTEventEmitter {
                               location: InAppLocation.from(number: locationNumber))
         }
     }
+    
+    @objc(getHtmlInAppContentForMessage:resolver:rejecter:)
+    func getHtmlInAppContent(messageId: String, resolver: RCTPromiseResolveBlock, rejecter: RCTPromiseRejectBlock) {
+        ITBInfo()
+        
+        guard let message = IterableAPI.inAppManager.getMessage(withId: messageId) else {
+            ITBError("Could not find message with id: \(messageId)")
+            rejecter("", "Could not find message with id: \(messageId)", nil)
+            return
```

Comment:
Not sure, but should we use rejecter to notify errors/failures?

Reply:
Yep https://reactnative.dev/docs/native-modules-ios.html#promises